src/main.ts

```diff
@@ -195,6 +196,31 @@ export default class ShikiPlugin extends Plugin {
 		}
 	}
 
+	registerInlineCodeProcessor(): void {
+		this.registerMarkdownPostProcessor(async (el, ctx) => {
+		    const inlineCodes = el.findAll(":not(pre) > code");
+		    for (let codeElm of inlineCodes) {
+		        let match = codeElm.textContent.match(/^\{([^\s]+)\} (.*)/i);  // format: `{lang} code`
+		        if (match && this.loadedLanguages.has(match[1])) {
```

Comment:
I don't think you need to check if the language exists here, as the highlight tokens method will return an empty array when there is no highlighter for the language.

Reply:
Actually it returns `undefined`, not an empty array.

```js
const highlight = await this.getHighlightTokens(match[2], match[1]);
const tokens = highlight ? highlight.tokens.flat(1) : [];
if (!tokens.length) continue;
```

Reply:
ah, then you could use optional chaining, but this seems fine as well